tools/bin/go_core_tests

```diff
@@ -4,8 +4,23 @@ set +e
 
 go test -v -p 4 -parallel 4 ./... >./output.txt
 EXITCODE=$?
-grep "\-\-\- FAIL" output.txt
-grep "FAIL	" output.txt
+
+echo "Failed tests and panics: ---------------------"
+echo ""
+grep --line-number --after-context=2 "panic: " output.txt
+echo ""
+grep --line-number "\-\-\- FAIL" output.txt
+grep -n "FAIL	" output.txt
+echo ""
+echo "----------------------------------------------"
+
+echo ""
+echo "(Note: panics abruptly end the test run and may appear out-of-place, not where they originate."
+echo "Finding their origin test may involve disabling parallelism)"
```

Comment:
This is my least fave thing about go's test suite. Absolutely horrible developer experience

Reply:
yup, no isolation between tests unfortunately